prusti-viper/src/encoder/procedure_encoder.rs

```diff
@@ -885,6 +885,7 @@ impl<'p, 'v: 'p, 'tcx: 'v> ProcedureEncoder<'p, 'v, 'tcx> {
                 vir::Type::Bool => BuiltinMethodKind::HavocBool,
                 vir::Type::TypedRef(_) => BuiltinMethodKind::HavocRef,
                 vir::Type::Domain(_) => BuiltinMethodKind::HavocRef,
+                vir::Type::Float => unreachable!("BackendType has no built-in methods?"),
```

Comment:
Here you probably need to have `BuiltinMethodKind::HavocF64` and `BuiltinMethodKind::HavocF32`.